genny-project/layouts
# Feature gen 1412

# Issue https://outcomelife.atlassian.net/browse/GEN-1412
**alyson-v3**: feature-gen-1412

# Description:
Added a tooltip component to facilitate easier testing with viewable test ids. Hovering the mouse over interact-able components such as buttons will show the test id at the top of the screen.

# How To Test
1) Log in to Internmatch with `?showcodes=true` as query parameters. ie: `http://localhost:3000/home?showcodes=true`
2) Hover mouse over tree items
3) Hover mouse over dropdown items
4) Hover mouse over buttons
5) Hover over table rows and buttons

When hovering over items, the testID should be should in the top center of the screen.
